README.md

```diff
@@ -1,7 +1,37 @@
-## One-time setup
+# Heath Equity Tracker
+
+The main codebase for [Health Equity Tracker](https://healthequitytracker.org/).
```

Comment:
This is actually the prototype and not the main codebase for HET. I'd leave this line out